build/Tests/FunctionalTests/FunctionalTests.vcxproj

```diff
@@ -295,8 +293,12 @@
   <ItemGroup>
     <Media Include="$(MSBuildThisFileDirectory)..\..\..\tests\functionaltests\Tests\AssetsLibraryTests\AssetsLibraryTestVideo.mp4" />
   </ItemGroup>
+  <ItemGroup>
+    <UXResources Include="$(MSBuildThisFileDirectory)..\..\..\samples\XAMLCatalog\XAMLCatalog\Images\*" />
```

Comment:
>UXResources [](start = 5, length = 11)

this makes me a little sad. Could we use ReferenceCopyLocalPaths instead and let the build just do its thing to copy?